step1_get_eia_demand_data.ipynb

```diff
@@ -163,43 +153,45 @@
     "        writer.writeheader()\n",
     "\n",
     "        full_date_range_dict = OrderedDict()\n",
+    "        datetime_format = '%Y-%m-%dT%H'\n",
     "        for hour in full_date_range:\n",
-    "            full_date_range_dict[hour.strftime(\"%Y%m%dT%HZ\")] = ['MISSING', 'MISSING']\n",
+    "            full_date_range_dict[hour.strftime(datetime_format)] = ['MISSING', 'MISSING']\n",
     "\n",
     "        # Actual realized demand\n",
-    "        for demand in region_data['series'][0]['data']:\n",
+    "        for demand in region_data['response']['data']:\n",
     "            # Skip dates outside the specified range\n",
-    "            if demand[0] not in full_date_range_dict.keys():\n",
+    "            if demand['period'] not in full_date_range_dict.keys():\n",
     "                continue\n",
     "            try:\n",
-    "                if demand[1] == None:\n",
-    "                    full_date_range_dict[demand[0]][0] = 'EMPTY'\n",
+    "                if demand['value'] == None:\n",
+    "                    full_date_range_dict[demand['period']][0] = 'EMPTY'\n",
     "                else:\n",
-    "                    full_date_range_dict[demand[0]][0] = demand[1]\n",
+    "                    full_date_range_dict[demand['period']][0] = demand['value']\n",
     "            except KeyError:\n",
-    "                print(\"Check date and time formatting for category {} for time {}\".format(region_id, demand[0]))\n",
+    "                print(\"Check date and time formatting for category {} for time {}\".format(region_id, demand['period']))\n",
     "\n",
     "        # Day ahead forecasted demand\n",
```

Comment:
Rm this forecast section?

Reply:
Are you seeing some BAs that have identical 2015 through 2020 data comparing the prior version and now? How many are changing historical values?

Reply:
yeh, I opened this PR before suggesting to remove the forecast data.

Haven't checked how many changed historical values, will do that!